app/controllers/Application.scala

```diff
@@ -29,7 +29,7 @@ class Application(
 
   private def applyCircles(intCmp: String, id: String, js: String, modifiedId: String, modifiedJs: String): (String, String) = {
     intCmp match {
-      case "gdnwb_copts_memco_sandc_circles_variant" => (modifiedId, modifiedJs)
+      case "true" => (modifiedId, modifiedJs)
```

Comment:
Will still be behind a flag?

Reply:
The current iteration of Circles on the bundles landing page is behind a flag, yes.